docs/how-to-guides/rancher-on-rancher-desktop.md

```diff
@@ -43,7 +43,7 @@ kubectl create namespace cert-manager
 
 4: Install cert-manager services:
 ```console
-helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v1.1.0 --set installCRDs=true
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v1.7.1 --set installCRDs=true
```

Comment:
Not specifying the version and let the helm command use the latest version works too.. For now, we can have the suggested `v1.7.1`.

Reply:
I guess, but watching rancher docs, looks like suggested version is 1.7.1 as of now